src/eligibility_matrix.js

```diff
@@ -116,7 +121,35 @@ const ELIGIBILITY_MATRIX = [
       work_at_least_1_year_1250_hours: 'yes',
       planned_c_section: 'no'
     },
-    eligibilities: ['high_earner']
+    eligibilities: ['high_earner', 'job_protected', 'pregnant_and_cfra']
+  },
+  {
+    label: 'CA W2 Pregnant CFRA C',
+    answers: {
+      why_need_time_off: 'pregnant',
+      confirm_state_ca: 'y',
+      which_state: undefined,

```

Comment:
FYI, leaving a question out of this list is the same as setting it to `undefined`.  For example, 
```
...
confirm_state_ca: 'y',
which_state: undefined,
w2_employee: 'y',
...
```
is the same as
```
...
confirm_state_ca: 'y',
w2_employee: 'y',
...
```

I don't know if leaving out things like `which_state` makes things clearer or more confusing...